dashboard/test/ui/features/step_definitions/steps.rb

```diff
@@ -1148,6 +1148,13 @@ def find_test_user_by_name(name)
   }
 end
 
+And /^I dismiss the teacher panel$/ do
+  steps %Q{
+    And I click selector ".fa-chevron-right"
+    And I wait until I see selector ".fa-chevron-left"
+  }
```

Comment:
Is there ever a possibility of another L/R chevron icon on that page? I can't think of when there would be but maybe adding a more specific identifier to the teacher panel icon for clarity and future proofing?

Reply:
Good point!  Refined in https://github.com/code-dot-org/code-dot-org/pull/36237.